packages/fluentui/react-northstar/src/components/Portal/usePortalBox.ts

```diff
@@ -25,15 +25,24 @@ export const usePortalBox = (options: UsePortalBoxOptions): HTMLDivElement | nul
   }, [target]);
 
   useIsomorphicLayoutEffect(() => {
+    const classes = className.split(' ').filter(Boolean);
+
     if (element) {
-      element.setAttribute('class', className);
+      element.classList.add(...classes);
 
       if (rtl) {
         element.setAttribute('dir', 'rtl');
       } else {
         element.removeAttribute('dir');
       }
     }
+
+    return () => {
+      if (element) {
+        element.classList.remove(...classes);
+        element.removeAttribute('dir');
+      }
+    };
```

Comment:
Previous behaviour didn't do this, are you just adding this now for correctness ?

Reply:
Adding now to be more consistent.